### DISCUSSION TITLE
bandwidth configuration for PC802R

### DISCUSSION THREAD

**Discussion by aibtw** [srsran/srsRAN_Project#661] [General Help]
Hi all, 

I checked the O-RU page for PC802 (https://docs.srsran.com/projects/project/en/latest/tutorials/source/oranRU/source/rus/picocom.html) and the configs in https://github.com/srsran/srsRAN_Project/blob/main/configs/gnb_ru_picocom_scb_tdd_n78_20mhz.yml

These gnb configs, is this just an example, or the actual testing configuration used by the srs team? 

Asking because I noticed that it is using 20MHz, but then it says "This configuration is relevant to firmware v3.0.0." and I when I checked in PC802R docs I saw that only 100MHz is supported for <RU5.x versions.

So, I just wanted to clarify if the used version is different or if there is a workaround or something I am missing.
Thanks.

**Comment by AlaiaL** [comment]
Hi Ahmed,

This example configuration was generated from the actual configuration used to test our gNB with Picocom SCB. With this configuration we were able to attach and do traffic. 
As the note indicates, this configuration is valid for SCB with  version RU3.0.0, we have not tested it with other versions. So answering your question, we tested with a different version. 

Thanks.
